src/libraries/System.IO.Ports/pkg/runtime.native.System.IO.Ports.props

```diff
@@ -13,6 +14,7 @@
     <NoWarn>$(NoWarn);NU5128</NoWarn>
     <!-- Disabling baseline validation since this is  binary blob -nd we do not need API compat. -->
     <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
+    <PackageReadmeFilePath>$(MSBuildThisFileDirectory)../src/PACKAGE.md</PackageReadmeFilePath>
```

Comment:
Why do we have a secondary definition of this property in a props file, besides the one in the csproj?

Reply:
The .props file is for all the runtime.RID packages (i.e. https://www.nuget.org/packages/runtime.osx-x64.runtime.native.System.IO.Ports/) and the proj one is the runtime.native project: https://www.nuget.org/packages/runtime.native.System.IO.Ports/9.0.0-preview.6.24327.7

Reply:
Okay good, so this ensures we have one readme to rule them all.